README.md

```diff
@@ -294,6 +294,32 @@ path.deploy_file(
 )
 ```
 
+[Deploy artifact by checksum](https://www.jfrog.com/confluence/display/RTF6X/Artifactory+REST+API#ArtifactoryRESTAPI-DeployArtifactbyChecksum): deploy an artifact to the specified destination by checking if the artifact
+content already exists in Artifactory. If Artifactory already contains a user
+readable artifact with the same checksum the artifact content is copied over to
+the new location without requiring content transfer.
+
+```python
+from artifactory import ArtifactoryPath
+
+path = ArtifactoryPath("http://my-artifactory/artifactory/my_repo/foo")
+sha1 = "1be5d2dbe52ddee96ef2d17d354e2be0a155a951"
```

Comment:
```suggestion
sha1 = artifactory.sha1sum("my_file.zip")
```

Reply:
Fixed

Reply:
@zhan9san 
have you pushed you changes?

Reply:
See below please